content/en/docs/tutorials/backup.md

```diff
@@ -49,5 +57,114 @@ created above after installing cert-manager.
 $ kubectl apply -f cert-manager-backup.yaml
 ```
 
-If you have migrated from an old cluster, you will need to make sure to run a
-similar `kubectl apply` command to restore your Secret resources too.
+## Full cluster backup and restore
+
+This section refers to backing up and restoring 'all' Kubernetes resources in a
+cluster- including some `cert-manager` ones- for scenarios such as disaster
+recovery, cluster migration etc.
+
+Note that we have not done any extensive testing, only a few common scenarios-
+it is recommended to test the backup and restore strategy before putting it in place
+as there are likely to be some other edge cases.
+We would like to ensure that users can reliably backup and restore `cert-manager`
+resources using common tools- if you encounter any errors, please do open a
+GitHub issue or PR an update to this document.
+
+### Avoiding unnecessary certificate reissuance
+
+#### Order of restore
+
+If `cert-manager` does not find a Kubernetes Secret with an X.509 certificate
+for a `Certificate`, reissuance will be triggered. To avoid unnecessary
+reissuance after a restore, ensure that Secrets are restored before
+`Certificate`s. Similarly, Secrets should be restored before `Ingress`es if you
+are using [`ingress-shim`](../../usage/ingress/).
```

Comment:
```suggestion
If `cert-manager` does not find a Kubernetes `Secret` with an X.509 certificate
for a `Certificate`, reissuance will be triggered. To avoid unnecessary
reissuance after a restore, ensure that `Secret`s are restored before
`Certificate`s. Similarly, `Secret`s should be restored before `Ingress`es if you
are using [`ingress-shim`](../../usage/ingress/).
```

Maybe quote `Secret` for consistency with the quoting of other Kubernetes resource types.